recipes/libftdi/1.x/patches/0001-cmake-targets.patch

```diff
@@ -0,0 +1,787 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 5aecafc..aa69995 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -1,36 +1,54 @@
+-# Project
+ project(libftdi1 C)
++
+ set(MAJOR_VERSION 1)
+ set(MINOR_VERSION 5)
+ set(PACKAGE libftdi1)
+ set(VERSION_STRING ${MAJOR_VERSION}.${MINOR_VERSION})
+ set(VERSION ${VERSION_STRING})
+-set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
++list(PREPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+ 
+-# CMake
+ if("${CMAKE_BUILD_TYPE}" STREQUAL "")
+    set(CMAKE_BUILD_TYPE     RelWithDebInfo)
+ endif("${CMAKE_BUILD_TYPE}" STREQUAL "")
+ set(CMAKE_COLOR_MAKEFILE ON)
+-cmake_minimum_required(VERSION 2.6 FATAL_ERROR)
++
++cmake_minimum_required(VERSION 3.0 FATAL_ERROR)
++if(POLICY CMP0057)
++    cmake_policy(SET CMP0057 NEW)
++endif()
++if(POLICY CMP0011)
++    cmake_policy(SET CMP0011 NEW)
++endif()
++if(POLICY CMP0099)
++    cmake_policy(SET CMP0099 NEW)
++endif()
```

Comment:
Please explain why you are adjusting the policies, these do not seem required

Reply:
If the policies are not set, there are warnings in the CMake log for recent versions.